src/sensors/tests/test_distant.py

```diff
@@ -218,8 +230,21 @@ def test_render(variant_scalar_rgb):
         "integrator": {"type": "path"}
     }
 
+    # set the sensor origin such that rays hit the square
+    if origin == {}:
+        pass
+    elif origin["origin"] == "disk":
+        origin["origin_center"] = [-w_o[i]*2 for i in range(3)]
+    elif origin["origin"] == "rectangle":
+        origin["origin_a"] = [-1-w_o[0]*2, -1-w_o[1]*2, 0-w_o[2]*2]
+        origin["origin_b"] = [1-w_o[0]*2, 1-w_o[1]*2, 0-w_o[2]*2]
+
+    dict_scene["sensor"] = {**dict_scene["sensor"], **origin}
     scene = load_dict(dict_scene)
     sensor = scene.sensors()[0]
     scene.integrator().render(scene, sensor)
     img = np.array(sensor.film().bitmap()).squeeze()
-    assert np.allclose(np.array(img), expected)
+    assert np.allclose(np.array(img), expected, rtol=1e-3)
```

Comment:
You can be stricter, there is no reason that our previous expectation should change.

Reply:
The tests run in single precision mode only:

`where False = <function allclose at 0x7f8256fd7a60>(array(0.6367, dtype=float16), (0.6366197505770148 * 1.0))`

As you can see, the computation result is rounded at the 1e-4 place already. I don't know how lower tolerance would work at all.

Reply:
Ok, so let's go with that then.